selfdrive/car/gm/interface.py

```diff
@@ -165,7 +166,9 @@ def _update(self, c):
 
       ret.buttonEvents = [be]
 
-    events = self.create_common_events(ret, pcm_enable=self.CP.pcmCruise)
+    events = self.create_common_events(ret, extra_gears=[GearShifter.sport, GearShifter.low,
+                                                         GearShifter.eco, GearShifter.manumatic],
```

Comment:
Before, these all would show up as `GearShifter.drive`?

Reply:
Correct

Reply:
I concur with Tim

Reply:
Since low and manumatic change the experience of the gas pedal, usually on other cars we block engaging here. Does it change engaged behavior or only disengaged behavior?